### ISSUE TITLE
[PATCH] add --pidfile '' for not creating pidfiles

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10360]
|<img alt="rgl's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rgl reported|
|-|-|
|Trac ID|trac#450|
|Type|enhancement|
|Created|2003-12-27 05:41:51Z|

Attachments:

* [nopid.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d96/d96adb142a1f7ede4d5906eadf01409dbd98b60d/971f5affe58feb18c9fcded59e5c11bcd746b080.patch) (2817 bytes) - added by rgl on 2003-12-27 05:41:51Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__450 450
type__enhancement enhancement
reporter__rgl rgl
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__core core
keywords__core core
time__1072503711000000 1072503711000000
changetime__1470055100583230 1470055100583230
version__ 
owner__moshez moshez
cc__exarkun cc__itamarst cc__moshez cc__rgl
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Assigning to moshez, who seems to be behind twistd
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
I'd like to hear why this is a useful feature.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

Apparently no one thinks this is a useful feature.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset cfdd18b2f1fa45b1901623cb46fbfb46be4524dc
```
#!CommitTicketReference repository="" revision="cfdd18b2f1fa45b1901623cb46fbfb46be4524dc"
Merge pull request #450 from twisted/8713-rodrigc-postfix

Author: rodrigc
Reviewer:  acabhishek942
Fixes: #8713

Convert twisted.protocols.postfix to Python 3
```
